next.config.js

```diff
@@ -35,6 +35,56 @@ const nextConfig = {
         permanent: true,
         basePath: false,
       },
+      {
+        source: "/server-side/v2/foundation/basics",
+        destination: "/server-side/v2/introduction/basics",
+        permanent: true
```

Comment:
set it true post going live and some dogfooding in production?

Reply:
makes sense, I've set it as false for now.